### DISCUSSION TITLE
selecting elements

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#139] [Q&A] [answered]
I can't remember if I've asked this before. How come sometimes when you select an element to work with you add the period before the element and sometimes you don't?

Example without the period before "header":

header {
	width: 100%;
	height: 50px;
	background-color: #1b1b32;
	dispay: flex;
}

**Comment by jakegros** [comment, marked as answer]
Yes, this confused me a lot, too.  It will still trip me up when I'm not paying close attention. 
I believe the period is used when it is a class selector. So, when you are selecting everything with that .class attribute. 

This has more examples: 
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors

**Reply by Skye404** [reply]
Thanks, this is helpful.
